app/assets/v2/js/pages/bounty_details.js

```diff
@@ -635,7 +635,7 @@ var attach_override_status = function() {
 
 var show_interest_modal = function() {
   var self = this;
-  let modals = $('#modalInterest');
+  var modals = $('#modalInterest');
```

Comment:
Can't we do this a `const` ? Is it changing somewhere or needed outside the block scope somehow?